packages/pieces/src/lib/apps/http/actions/send-http-request-action.ts

```diff
@@ -0,0 +1,50 @@
+import { assertNotNullOrUndefined } from "../../../common/helpers/assertions";
+import { httpClient } from "../../../common/http/core/http-client";
+import { HttpHeaders } from "../../../common/http/core/http-headers";
+import { HttpRequest } from "../../../common/http/core/http-request";
+import { QueryParams } from "../../../common/http/core/query-params";
+import { createAction } from "../../../framework/action/action";
+import { Property } from "../../../framework/property";
+import { httpMethodDropdown } from "../common/props";
+
+export const httpSendRequestAction = createAction({
+  name: 'send_request',
+  displayName: 'Send HTTP request',
+  description: 'Send HTTP request',
+  props: {
+    method: httpMethodDropdown,
+    url: Property.ShortText({
+      displayName: 'URL',
+      required: true,
+    }),
+    headers: Property.Object({
+      displayName: 'Headers',
+      required: false,
```

Comment:
I think they should be required, if they didn't get filled, they will be an empty object anyway